meson.build

```diff
@@ -231,3 +243,30 @@ gnome.post_install(
     update_desktop_database: true,
     gtk_update_icon_cache: true,
 )
+
+if host_machine.system() == 'darwin' and get_option('build_macos_app').enabled()
+    app_macos = app_contents / 'MacOS'
+    app_resources = app_contents / 'Resources'
+
+    install_data(
+        'macos/launcher.sh',
+        rename: 'gtkwave',
+        install_dir: app_macos,
+        install_mode: 'rwxr-xr-x'
+    )
+
+    install_data(
+        'macos/Info.plist',
+        install_dir: app_contents,
+    )
+
+    install_data(
+        'macos/AppIcon.icns',
+        install_dir: app_resources
+    )
+
+    install_data(
+        'macos/gtkwave.entitlements',
+        install_dir: app_contents
+    )
```

Comment:
Please rename the `app_...` variables in something like `macos_...`. All variables in meson are global and that will future variable name clashes less likely.